dockers/docker-database/base_image_files/redis-cli

```diff
@@ -1,10 +1,15 @@
 #!/bin/bash
```

Comment:
This script is to mimic redis-cli behavior inside database container. Please don't add new feature here.

Reply:
Fixed, added a new wrapper sonic-db-cli to invoke sonic-db-cli in specific namespace

Reply:
Could we still use sonic-db-cli script it self without adding new wrapper.
1. new wrapper will make the performance worse
2. the new wrapper looks only installed on host, how about docker environment, still using sonic-db-cli comes with swsssdk lib? They are not identical.

@qiluo-msft  please share your opinion as well

Reply:
@dzhangalibaba : sonic-db-cli wrapper is similar to the older redis-cli wrapper which was executing command inside database docker. There is a parallel activity to include multi-DB changes. Once that is complete, sonic-db-cli should be able to talk to a specific DB instance without using this wrapper. This PR is specifically for bringing up multi-asic platform. Combining multi-db and multi-asic changes together can be cumbersome to review and test. After multi-DB changes are in, this wrapper will not be required.